.github/workflows/continuous-integration.yml

```diff
@@ -0,0 +1,81 @@
+name: "Continuous Integration"
+
+on:
+  pull_request:
+    branches:
+      - "*.x"
+      - "master"
+  push:
+    branches:
+      - "*.x"
+      - "master"
+
+env:
+  fail-fast: true
+
+jobs:
+  phpunit:
+    name: "PHPUnit"
+    runs-on: "ubuntu-20.04"
+
+    strategy:
+      matrix:
+        php-version:
+          - "7.1"
+          - "7.2"
+          - "7.3"
+          - "7.4"
```

Comment:
PHP 8 is not tested anymore, and will be the source of a lot more changes I'm afraid… You will have to upgrade `doctrine/coding-standard` to v8, maybe also PHPStan.

Reply:
You can resurrect the Travis file and leave only the `nightly` job if you want to do this in a separate PR, it's as you wish.

Reply:
Oh but I forgot: Travis no longer runs :facepalm:

Reply:
> PHP 8 is not tested anymore, and will be the source of a lot more changes I'm afraid… You will have to upgrade doctrine/coding-standard to v8, maybe also PHPStan.

Yeah I already ran into some of those issues when I started working on #33. I have PHP 8.0 installed on my local machine, so at least I can work on this without waiting for CI all the time. I will continue my work when this PR is merged.